### PR TITLE
Feat/blog

### PR DESCRIPTION
**Pull request by lorenzolewis**
@nothingismagick 

Remaining items:
- Didn't get to proof-read everything yet
- Final quote section I didn't get to trim down, so have at it
- The Accolades section doesn't have the images yet

### CODE REVIEW

File: blog/tauri_1_0.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+| @emirror-de       | System tray support                                                                              | tao                                 |
+| @lorenzolewis     | tauri.app updates                                                                                | tauri-docs                          |
+| @probablykasper   | Support for more accelerators, restructured documentation                                        | tao, tauri-docs                     |
+| @grbd             | Added an example to use tauri as a C++ DLL                                                       | tauri                               |
+| @youngsing        | Added macOS vibrancy                                                                             | tauri-plugin-vibrancy               |
+| @lemarier         | Updater, menus, system tray, iOS, clipboard api, bytes-stream & ++                               | wry, tao, tauri                     |
+| @JonasKruckenberg | Tauri plugin upgrades                                                                            | tauri-plugin-\*                     |
```
Comment (JonasKruckenberg) [inline blog/tauri_1_0.md]:
```suggestion
| @JonasKruckenberg | Tauri plugin upgrades and documentation overhaul                                                                          | tauri-plugin-\* tauri-docs                     |
```

Suggesting this feels weird, but so did my previous contribution text sooo 🤷🏻‍♂️

File: blog/tauri_1_0.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+![Johannes Schickling at Worker Conf 2022 (photo by @TejasKumar)](./tauri_1_0_images/worker_conf_2022.png)
+
+<figcaption>Johannes Schickling at Worker Conf 2022 (photo by <a href="https://twitter.com/TejasKumar_">@TejasKumar via Twitter</a>)</figcaption>
+
+</figure>
+
+### Environment
+
+The apps you make are [lean and performant](https://tauri.app/about/benchmarks), which reduces electricity, storage space, and general natural resource consumption. Every byte saved is a leaf on a tree that gets to grow.
+
```
Comment (JonasKruckenberg) [inline blog/tauri_1_0.md]:
```suggestion
To illustrate this, we compiled some numbers on the ecological impact of your apps size. As you can see, even small increases in size have a hefty impact on the environment!
```

File: blog/tauri_1_0.md (hunk 1 above)
Comment (lorenzolewis) [reply blog/tauri_1_0.md]: Added this in on my local copy

File: blog/tauri_1_0.md (hunk 2 above)
Comment (lorenzolewis) [reply blog/tauri_1_0.md]: Added locally

File: blog/tauri_1_0.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+<figcaption><i>Johannes Schickling at Worker Conf 2022 (photo by <a href="https://twitter.com/TejasKumar_">@TejasKumar via Twitter</a>)</i></figcaption>
+
+</figure>
+
+### Environment
+
+The apps you make are [lean and performant](https://tauri.app/about/benchmarks), which reduces electricity, storage space, and general natural resource consumption. Every byte saved is a leaf on a tree that gets to grow.
+
+To illustrate this, we compiled some numbers on the ecological impact of your apps size. As you can see, even small increases in size have a hefty impact on the environment
```
Comment (JonasKruckenberg) [inline blog/tauri_1_0.md]:
```suggestion
To illustrate this, we compiled some numbers on the ecological impact of your apps size. As you can see, even small increases in size have a hefty impact on the environment!
```

**Review by nothingismagick — APPROVED** [review]

**Pull request merged**